Yoast/wordpress-seo
# Still show old metabox on flag

## Summary

This PR can be summarized in the following changelog entry:

* Stopped replacing metabox content on enabled feature flag.

## Relevant technical choices:

* Reverted #10364 for the most part.

## Test instructions

This PR can be tested by following these steps:

* Checkout this branch;
* Make sure you have the feature flag enabled: `define( 'YOAST_FEATURE_GUTENBERG_SIDEBAR', true );
`
* Check if the content tab of the metabox displays correctly..
## Quality assurance


Fixes #10427
